app/upgrades/v4/upgrade.go

```diff
@@ -144,7 +145,7 @@ func ConvertOldRollappToNew(oldRollapp rollapptypes.Rollapp) rollapptypes.Rollap
 		Frozen:           oldRollapp.Frozen,
 		RegisteredDenoms: oldRollapp.RegisteredDenoms,
 		// TODO: regarding missing data - https://github.com/dymensionxyz/dymension/issues/986
-		Bech32Prefix:    bech32Prefix,                                        // placeholder data
+		Bech32Prefix:    oldRollapp.RollappId[:5],                            // placeholder data
```

Comment:
I know it's same as the old code but do you know why 5?

Reply:
no idea actually, saw it as well 😄 
probably will be changed on 
```
// TODO: regarding missing data - https://github.com/dymensionxyz/dymension/issues/986
```